.coveragerc

```diff
@@ -0,0 +1,2 @@
+[run]
+branch = True
```

Comment:
Yes we can drop branch coverage.  It is not required by codecov.  Adding branch coverage to everything is my personal goal because I don't like to see [missing else branches][] go unnoticed.  I'll remove the `.coveragerc` file for now.

[missing else branches]: https://codecov.io/github/treyhunner/django-rest-framework/rest_framework/fields.py?ref=48b21b6063fec77fc938f9ee7eb41f0d147dc1d1#l-1435